RoleplayingSchemaBackend/Migrations/RoleplayingDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,50 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using RoleplayingSchemaBackend.Data;
+
+#nullable disable
+
+namespace RoleplayingSchemaBackend.Migrations
+{
+    [DbContext(typeof(RoleplayingDbContext))]
+    partial class RoleplayingDbContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
+            modelBuilder
+                .HasAnnotation("ProductVersion", "7.0.13")
+                .HasAnnotation("Relational:MaxIdentifierLength", 128);
+
+            SqlServerModelBuilderExtensions.UseIdentityColumns(modelBuilder);
+
+            modelBuilder.Entity("RoleplayingSchemaBackend.Data.Users", b =>
+                {
+                    b.Property<Guid>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("uniqueidentifier");
+
+                    b.Property<string>("Email")
+                        .IsRequired()
+                        .HasColumnType("nvarchar(max)");
```

Comment:
Is the email meant to be required? We talked about it being optional, so I think this might be by mistake? 

If it is a mistake it should probably be fixed by changing the email type to a nullable string and recreate the migration instead of changing it here.

Reply:
I actually have 0 Clue as to why it did that, there in no way is anywhere, where that should be required I have used 0 annotations to make that requirement, but well spotted !